src/link.cpp

```diff
@@ -1043,6 +1043,17 @@ static void construct_linker_job_elf(LinkJob *lj) {
                 lj->args.append("-lm");
                 lj->args.append("--end-group");
             } else {
+                if (g->zig_target->os == OsDragonFly) {
+                    // path to libgcc.a
+                    lj->args.append(
+                        "-L/raven/share/raven/toolchain/gcc8/lib/gcc/x86_64-raven-dragonfly5.4/8.3.0");
+                    // path to libgcc_s.so
+                    lj->args.append(
+                        "-L/raven/share/raven/toolchain/gcc8/lib");
+                    // prefer .a over .so
+                    // fixes: lld: warning: found local symbol '__thread_locale' in global part of symbol table in file /raven/share/raven/sysroot/DragonFly/usr/lib/libc.so
+                    lj->args.append("/raven/share/raven/sysroot/DragonFly/usr/lib/libc.a");
```

Comment:
Not a blocker, just a question from an interested onlooker: how would someone who uses dports rather than ravenports use zig on dragonflybsd?

Reply:
ravenports should work the same on linux/macos/solaris/freebsd/dragonfly and maybe netbsd. It looks like they will accept my port, if so, the command to install it will be
```
> /raven/sbin/pkg install zig-single-standard
```

And from there on it should be the same as dports -- though using /raven/lib etc rather than /usr/lib etc